db/seeds.rb

```diff
@@ -1,3 +1,4 @@
 # frozen_string_literal: true
 
 FactoryBot.create_list(:user, 6)
+FactoryBot.create_list(:message, 10)
```

Comment:
This will create additional user for every message, which is probably not what you want. You can reuse the users created one line above and pass them to a `message` factory.

Reply:
The return value of a `.create_list` is a list of instantiated `User` models. You can then use them for creating the messages for existing users by passing `user:` arg into `message` factory.

This way you can make new messages be assigned to users you've just created. Or, if you really want your seeds to be truly "rich", you can even create multiple clauses - a user with no messages, a user with multiple messages, etc.